tools/rimage/src/include/rimage/cavs/cavs_ext_manifest.h

```diff
@@ -27,7 +27,7 @@
 #define EXTENDED_MANIFEST_VERSION_MAJOR		0x0001
 #define EXTENDED_MANIFEST_VERSION_MINOR		0x0000
 
-#define FW_MAX_EXT_MODULE_NUM	32
+#define FW_MAX_EXT_MODULE_NUM	48
```

Comment:
What's the container size ? Lets just set this unlimited.

Reply:
@lgirdwood it is used for a fixed-size array inside a structure. So the only alternative would be to count and dynamically allocate

Reply:
ack, I think we need the dynamic alloc (with a sane max). Whats the array elem size ?

Reply:
@lgirdwood one element is 48 bytes large (if I counted correctly). Since this is rimage, running on a build machine with gigabytes of RAM, maybe this optimisation isn't very urgent and can be postponed to a later PR?

Reply:
ack - that's fine - no impact to FW foot print. Since its host I would make it high in a followup PR to future proof rimage for v2.13